VideoLocker/src/main/java/org/edx/mobile/model/IVertical.java

```diff
@@ -0,0 +1,13 @@
+package org.edx.mobile.model;
+
+
+import java.util.List;
+
+public interface IVertical extends IComponent{
```

Comment:
Is anything going to implement IVertical besides VerticalModel? Not sure having the separate interface is getting us anything? Same for all these other interfaces with one implementation.